packages/auth/src/platform_cordova/popup_redirect/events.test.ts

```diff
@@ -73,28 +70,28 @@ describe('platform_cordova/popup_redirect/events', () => {
 
   describe('_savePartialEvent', () => {
     it('sets the event', async () => {
+      const spy = sinon.spy(Storage.prototype, 'setItem');
```

Comment:
Can you explain why there is a change from stub to spy?

Reply:
I was having issues with the stub at one point and the test only required verifying if the method was called so stubbing out a return value was not necessary for the test itself.